pkg/apis/operator/v1alpha1/types.go

```diff
@@ -63,6 +64,8 @@ type GardenSpec struct {
 
 // RuntimeCluster contains configuration for the runtime cluster.
 type RuntimeCluster struct {
+	// Ingress configures Ingress specific settings for the Garden cluster. This field is immutable.
+	Ingress *Ingress `json:"ingress"`
```

Comment:
If it's not optional it should not be a pointer
```suggestion
	Ingress Ingress `json:"ingress"`
```